config.py

```
@@ -0,0 +1,285 @@
+EXPECTED_DEV_BOT_ID: str = os.environ.get("EXPECTED_DEV_BOT_ID", "").strip()
+
+
+def _extract_bot_id(token: str) -> str:
+    """Return the numeric bot-ID prefix of a Telegram bot token, or ''."""
+    if not token or ":" not in token:
+        return ""
+    bot_id, _ = token.split(":", 1)
+    return bot_id if bot_id.isdigit() else ""
+
+
+def validate_config() -> None:
```

Ruff (0.15.7)

[warning] Too many branches (18 > 12)

(PLR0912)